ui/root/src/commonMain/kotlin/social/androiddev/root/splash/SplashContent.kt

```diff
@@ -27,9 +27,11 @@ import social.androiddev.root.navigation.SplashComponent
 @Composable
 fun SplashContent(
     component: SplashComponent,
+    authStatus: AuthStatus?,
```

Comment:
the UI/View _should not_ know about or be coupled with `domain` level objects. Please map domain -> UI model if needed